includes/Checker/Checks/Enqueued_Resources_Check.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Class WordPress\Plugin_Check\Checker\Checks\Enqueued_Resources_Check
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check\Checker\Checks;
+
+use WordPress\Plugin_Check\Checker\Check_Categories;
+use WordPress\Plugin_Check\Traits\Stable_Check;
+
+/**
+ * Check for running WordPress enqueued resources sniffs.
+ *
+ * @since n.e.x.t
+ */
+class Enqueued_Resources_Check extends Abstract_PHP_CodeSniffer_Check {
+
+	use Stable_Check;
+
+	/**
+	 * Gets the categories for the check.
+	 *
+	 * Every check must have at least one category.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array The categories for the check.
+	 */
+	public function get_categories() {
+		return array( Check_Categories::CATEGORY_PERFORMANCE );
```

Comment:
Not sure about this. Whether or not you hard-code assets doesn't really do anything to performance. I think this is rather a general best practice or plugin requirement.

Reply:
Which category is preferred here? I will update the PR when we get confirmation about warning/error from the review team.

Reply:
I go with `General` category.

Reply:
Hello, It has to be in repo category. It's a requirement for the plugins that we review.

Reply:
@davidperezgar This has been already updated.

Reply:
Ok